src/Vts.Scripting/ForwardSolvers/Demo04FluenceOfRhoAndZAndFt.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Vts.Common;
+using Vts.Modeling.ForwardSolvers;
+using Plotly.NET.CSharp;
+using System;
```

Comment:
Another unused using, maybe remove these globally in the scripting project (to avoid many file changes in case we still have them in VTS).

I think when you have implicit usings set to true, you no longer need to include these main usings.

Reply:
That's a good idea, thanks. In fact, it's likely the notebooks will have a "header" that will bring in all needed usings, and the Implcit usings + GlobalUsings will mimic that best.